solr/core/src/java/org/apache/solr/core/CloudConfig.java

```diff
@@ -50,6 +50,8 @@
 
   private final String pkiHandlerPublicKeyPath;
 
+  public static final int DEFAULT_AUTO_REPLICA_FAILOVER_WAIT_AFTER_EXPIRATION = 120000;
```

Comment:
Let's move this to OverseerTriggerThread because that's the only class which uses this constant. Maybe rename to `DEFAULT_AUTO_ADD_REPLICA_WAIT_FOR_SECONDS` and set it to `120`.